src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -160,6 +160,19 @@
 				@lightbox-closed="checkoutLightboxClosed"
 				title="Checkout"
 			>
+				<campaign-status
+					v-if="!hideStatusBar && activeCreditType === 'lending_reward'"
+					class="corporate-campaign-landing__status--incontext collapse"
```

Comment:
Since 
`corporate-campaign-landing__status--incontext`
is not modifying a
`corporate-campaign-landing__status` 
class, I'd recommend using a regular element syntax like `corporate-campaign-landing__incontext-status`

Reply:
Will give it a try. Need to make sure it overrides the previous `corporate-campaign-landing__status` values.